backend/lib/calculate/adequacy.py

```diff
@@ -11,8 +12,7 @@
 
 from sqlalchemy.orm import sessionmaker
 
-# TODO - Use config or environment variable.
-MEASURER = distance.get_measure('haversine')
+MEASURER = distance.get_measure(config.get('measurer'))
 EXIT_DISTANCE = 10.0    # Measured in miles.
 RELEVANCY_RADIUS_IN_METERS = 24140.2    # 15 miles in meters. Do not go lower than this.
 NUM_ADEQUACY_PROCESSORS = 8
```

Comment:
Move to config.

Reply:
This has not changed in this PR. Also, config did not exist when this change was introduced :)

Reply:
But now it does  :)

Reply:
Done.